src/main/java/uk/gov/hmcts/reform/professionalapi/controller/constants/ProfessionalApiConstants.java

```diff
@@ -132,6 +132,8 @@ private ProfessionalApiConstants() {
     public static final String GET_USER_STATUS_EMAIL_1 = "**IDAM Roles to access API** : \n pui-finance-manager";
     public static final String GET_USER_STATUS_EMAIL_2 = ",\n pui-user-manager,\n pui-organisation-manager,";
     public static final String GET_USER_STATUS_EMAIL_3 = "\n pui-case-manager,\n caseworker-publiclaw-courtadmin";
-
+    public static final String UUID_PATTERN = "^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-"
```

Comment:
This regex seems like a really bad idea. Do you really need to validate UUIDs at all? 
I'm not sure that, even if you do really really want to do this, the regex is correct. 
This is a valid type 1 UUID:
76f2854a-3b4c-11ed-a261-0242ac120002
Your regex states that the 3rd group of chars has to begin with [089ab]. It's possible that this is correct, because that part of the UUID does encode the version, but I'm not convinced it's ref data's business to be policing this. 
Also you should allow upper and lower case alpha values in hex.

Reply:
Thank you @andywilkinshmcts this issue has been addressed Request you to please review ...

Reply:
Thanks @andywilkinshmcts removed the validation of UUID Regex and limited its validation in terms of empty value ...